other/README.md

```diff
@@ -0,0 +1,45 @@
+# Master Links
+
+Important and useful :link: links to other documentation and standards at NYPL.
+
+## SimplyE
+
+* [Main "meta-repository"](https://github.com/NYPL-Simplified/Simplified) and [Wiki](https://github.com/NYPL-Simplified/Simplified/wiki)
+* [Circulation Manager](https://circulation.librarysimplified.org)
+
+## Search
+
+* [Search Analytics](https://github.com/NYPL/search-analytics)
+
+## Data
+
+* [Data Repo](https://github.com/NYPL/nypl-core)
+  * Models, mappings, and vocabularies for the NYPL Core ontology.
+
+## Platform
+
+* [Platform API Documentation](https://docs.google.com/document/d/1p3q9OT9latXqON20WDh4CNPxIShUunfGgqT163r-Caw/edit?usp=sharing)
+* [OAuth Server Documentation](https://confluence.nypl.org/display/WT/ISSO)
+
+## Data Warehouse
+
+* [Data Warehouse Documentation](https://github.com/NYPL/data-warehouse)
+
+## Shared Collection Catalog/ReCAP
+
+* [Shared Collection Catalog UI](https://www.nypl.org/research/collections/shared-collection-catalog/)
+* [“Best Of” ReCAP For Developers](https://docs.google.com/document/d/1-bZsN4sRay2Vnnov-4t_D56kBnIFbyQWr8U0ZGXdUso)
+* [Request Architecture](https://docs.google.com/presentation/d/1Tmb53yOUett1TLclwkUWa-14EOG9dujAyMdLzXOdOVc)
```

Comment:
I would add the CancelRequest architecture as well.

Reply:
Please feel free to add it. I think better to have more stuff than less... we can reorganize as necessary. Thanks!

Reply:
I added it.

I've also added a section `Web Applications` in *Other* (maybe there is a better place for this... for now I will leave as is) that lists all of the SPA's written by framework React or Angular.

It is easy to get lost in what app belongs to what repo and what is the URL. Hopefully, we can keep this list up to date when we create/deprecate applications.

@EdwinGuzman -- If I missed any App, please add or let me know.

CC: @kfriedman

Reply:
You could add the Header app since that's a React app.